Package@swift-5.6.swift

```diff
@@ -0,0 +1,146 @@
+// swift-tools-version:5.6
```

Comment:
happy to also switch to the other style (file per "old" tools-version), lets finalize the discussion inhttps://github.com/apple/swift-statsd-client/pull/36#issuecomment-1205800362 and then apply everywhere

Reply:
@Lukasa ^^